### PR TITLE
Update working-with-user-defined-types-in-sql-server.md

### PR DESCRIPTION
**Pull request by alberto-morillo**
When we read the title of this page we expect to find examples of how to use user-defined data types with SQL Server.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@alberto-morillo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by rothja** [comment]
@alberto-morillo Alberto, thanks for taking the time to create a new example! This is great. I need a little more time to test out your sample and get back to you on the change. Thanks for your patience!

**Comment by MikeRayMSFT** [comment]
@alberto-morillo - I reviewed this with @uc-msft 

Would you mind modifying your example to not use cursor but instead execute a SELECT statement computing say MIN/MAX of the integer values. 

Thank you,
Mike

File: docs/relational-databases/clr-integration-database-objects-user-defined-types/working-with-user-defined-types-in-sql-server.md [hunk 1]
```diff
@@ -21,6 +21,49 @@ ms.author: "jroth"
...
+    WHILE @@FETCH_STATUS = 0
+    BEGIN
+
+       PRINT 'Int var is : ' + CONVERT(VARCHAR(max),@myInt);
+
+       FETCH NEXT FROM intListCursor INTO @myInt;
+    END;
+
+    CLOSE intListCursor;
+    DEALLOCATE intListCursor;
```
Comment (MikeRayMSFT) [inline docs/relational-databases/clr-integration-database-objects-user-defined-types/working-with-user-defined-types-in-sql-server.md:60]:
```suggestion
    DEALLOCATE intListCursor;

```

**Comment by rothja** [comment]
@alberto-morillo I think Mike Ray has a good point about the sample being stronger if we avoided cursors. Having not heard back from you on this, I am going to close the public PR for now, but if you want to look at the suggestion and propose a modified sample, I can help test and publish it. We can reopen this PR and continue working at any time. Thanks!

**Pull request closed without merging**